### ISSUE TITLE
Sync hidden blog status with WordPress.com

### ISSUE THREAD

**Issue by koke** [wordpress-mobile/WordPress-iOS#969]
From #715, let's sync blog visibility with WordPress.com once the API for it is in place.

Moving to a separate ticket since this will be punted to 3.9.1 or later

**Comment by folletto** [comment]
> Can you point me in the right direction?

From the [console](https://developer.wordpress.com/docs/api/console/), isn't it the `"visible": true` parameter returned by the API? :)

**Comment by ebinnion** [comment]
FWIW - It looks like we can also pass `site_visibility=visible` in the URL to get only the visible sites for a user.

![screen shot](https://cloud.githubusercontent.com/assets/1126811/6397385/ec621c1a-bda9-11e4-812f-972284fcee7c.png)

**Comment by koke** [comment]
Oh, I see it know. What I'm missing is the `POST` part of that API. Since you can hide blogs from the app, the sync needs to be 2-way
